fiaas_skipper/deploy/bootstrap.py

```diff
@@ -17,38 +18,40 @@
 
 
 class BarePodBootstrapper(object):
-    def __init__(self, cmd_args=()):
+    def __init__(self, cmd_args=[]):
```

Comment:
Using list here is a bad choice. Mutable objects should not be the default value in a parameter-list, it sets the stage for hard to understand bugs later. If this really needs to be a list, it should default to `None`, with initializing in the `__init__`-method.

Reply:
You're right, good catch! It is used in the `command` field of the bootstrapping pod/spec/container, which is a `ListField`, so it needs to be a list at that point. I'll set the argument default to None and initialize it in the init method. 
This only worked because the production code paths that use this always set `cmd_args` (to a list), and it started breaking in the tests I wrote when I didn't.